src/actors/symbolication.rs

```diff
@@ -671,6 +672,10 @@ impl SymCacheLookup {
                 if let Some(ref symcache) = symcache {
                     entry.object_info.arch = symcache.arch();
                     entry.object_info.features.merge(symcache.features());
+
+                    let mut difs = Vec::new();
+                    difs.extend_from_slice(&symcache.candidates());
+                    entry.object_info.candidates = difs; // TODO(flub): merge!
```

Comment:
Not sure what this comment is about, but can’t you just `extend_from_slice` on the `object_info.candidates`?

Reply:
That comment is for future work, when the info is also added for CFI objects as well we have to be more careful rather than just extend a vec.  This already improves in the followup PR (#316)